src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -52,14 +75,36 @@ public UserTenant getById(UUID consortiumId, UUID id) {
     return converter.convert(userTenantEntity, UserTenant.class);
   }
 
+  @Override
+  public UserTenant save(UUID consortiumId, UserTenant userTenantDto) {
+    FolioExecutionContext getContext = (FolioExecutionContext) folioExecutionContext.getInstance();
```

Comment:
maybe rename to smth like currentTenantContext?

Reply:
fixed